views/canvas.ejs

```diff
@@ -2,25 +2,55 @@
 <html>
 
   <head>
-    <title>Running Lean | <%= title %></title>
-    <link rel='stylesheet' href='/stylesheets/style.css' />
+    <title>Running Lean | <%= title %> </title>
+    <link rel='stylesheet' href='/stylesheets/base.css' />
+    <link rel='stylesheet' href='/stylesheets/canvas.css' />
   </head>
 
   <body>
```

Comment:
So is there no "channels" or "key metrics"?

What about an input to modify the title?